.vscode/settings.json

```diff
@@ -9,5 +9,6 @@
     "**/versions/[0-9]**/**": true
   },
   "prettier": false,
+  "editor.formatOnSave": false,
```

Comment:
Looks like this is already the default setting.

Reply:
it depends on user preferences, in our mono repo projects I have my set to auto format and lint everything...

```
{
  "editor.formatOnSave": true,
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  },
  "eslint.validate": ["javascript", "typescript"],
  "eslint.workingDirectories": [
      {
          "pattern": "packages/*"
      }
  ]
}

```

This reduces the amount of `chore: lint fix` commits. I can remove the explicit disable if you like.

Reply:
Don't feel strongly on this one. Call is yours.